tests/api.c

```diff
@@ -19134,6 +19134,7 @@ static int test_wc_curve25519_export_key_raw(void)
     ExpectIntEQ(wc_curve25519_export_key_raw(&key, privateKey, &prvkSz,
         publicKey, NULL), BAD_FUNC_ARG);
 
+<<<<<<< HEAD
```

Comment:
There are merge artifacts that need to be resolved. This is why the tests are failing.

Reply:
Ah, sorry about that. Should be fixed.